release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1438,7 +1450,7 @@ This release removes the Prometheus Adapter, which was a Technology Preview feat
 
 * Previously, for virtual machines that have a `RerunOnFailure` run strategy, stopping the virtual machine resulted in several UI elements becoming unresponsive, preventing users from reading status information or restarting the virtual machines. This update fixes the unresponsive elements so users can use those features. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1951209[*BZ#1951209*])
 
-* Previously, for clusters that were configured to have a separate `/var` partition, querying the file system returned the size only of disks mounted in the root directory, excluding the size of the `/var` partition. This fix changes how that query runs, and users can now determine the total size of the file system on a cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1960612[*BZ#1960612*]))
+* Previously, for clusters that were configured to have a separate `/var` partition, querying the file system returned the size only of disks mounted in the root directory, excluding the size of the `/var` partition. This fix changes how that query runs, and users can now determine the total size of the file system on a cluster. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1960612[*BZ#1960612*])
```

Comment:
I know this was already merged, so this is just a suggested edit: 

"the file system only returned the size of disks mounted in the..."

Instead of:
"the file system returned the size only of disks mounted in the..."